doc/api/tty.md

```diff
@@ -63,7 +63,9 @@ added: v0.7.7
 * `mode` {boolean} If `true`, configures the `tty.ReadStream` to operate as a
   raw device. If `false`, configures the `tty.ReadStream` to operate in its
   default mode. The `readStream.isRaw` property will be set to the resulting
-  mode.
+  mode. 
+  
+  Returns {this} - the read stream instance.
```

Comment:
s/Returns/Returns:/

Reply:
Sorry, I did not explain: our docs are parsed with tools (for example, a converter into JSON), so some format unification is needed. This should be literally:
```md
* Returns: {this} - the read stream instance.
```

Reply:
yeah I wasn't sure, will fix, I get it